### ISSUE TITLE
Confusing error output on deploy to nodejitsu that shouldn't be there

### ISSUE THREAD

**Issue by leejt489** [balderdashy/sails#2443]
When I deploy in nodejitsu I get this error: `"In model (appliancetype), invalid connection :: mongoAlmanacProd Must contain an `adapter` key referencing the adapter to use."`

And then it just continues for a while until my app starts and then eventually stop.  Someone deploying in production with the log silent might never see it, but it screwed me up for a good while because it distracted me from the real error that was preventing my app from starting

Nothing special about that model "appliancetype", except that it is the first alphabetically.  That connection definitely has an adapter key, everything was great in my local environment, and once I tracked down the other error and got it up on nodejitsu, the app seems to be working there.

Here is what the log looks like:

err Wed, 26 Nov 2014 11:48:17 GMT error: Must contain an `adapter` key referencing the adapter to use.
err Wed, 26 Nov 2014 11:48:28 GMT error: In model (appliancetype), invalid connection :: mongoAlmanacProd
err Wed, 26 Nov 2014 11:48:28 GMT error: Must contain an `adapter` key referencing the adapter to use.
err Wed, 26 Nov 2014 11:48:38 GMT error: In model (appliancetype), invalid connection :: mongoAlmanacProd
err Wed, 26 Nov 2014 11:48:38 GMT error: Must contain an `adapter` key referencing the adapter to use.
err Wed, 26 Nov 2014 11:48:49 GMT error: In model (appliancetype), invalid connection :: mongoAlmanacProd
err Wed, 26 Nov 2014 11:48:49 GMT error: Must contain an `adapter` key referencing the adapter to use.
err Wed, 26 Nov 2014 11:49:00 GMT error: In model (appliancetype), invalid connection :: mongoAlmanacProd
error: Must contain an `adapter` key referencing the adapter to use.
err Wed, 26 Nov 2014 11:49:08 GMT error: In model (appliancetype), invalid connection :: mongoAlmanacProd
error: Must contain an `adapter` key referencing the adapter to use.
out Wed, 26 Nov 2014 11:49:16 GMT info: Creating API Docs...
err Wed, 26 Nov 2014 11:49:17 GMT error: In model (appliancetype), invalid connection :: mongoAlmanacProd
err Wed, 26 Nov 2014 11:49:17 GMT error: Must contain an `adapter` key referencing the adapter to use.
out Wed, 26 Nov 2014 11:50:07 GMT info: ...finished creating API Docs
out Wed, 26 Nov 2014 11:50:07 GMT info: 
out Wed, 26 Nov 2014 11:50:07 GMT info: ____---___--___---___--___---___--___-__
out Wed, 26 Nov 2014 11:50:07 GMT info: __---___--___---___--___---___--___
out Wed, 26 Nov 2014 11:50:07 GMT info: Sails <|
out Wed, 26 Nov 2014 11:50:07 GMT info: / || \
out Wed, 26 Nov 2014 11:50:07 GMT info: Finished bootstrap jobs! Start using Almanac!
err Wed, 26 Nov 2014 11:50:07 GMT debug: :: Wed Nov 26 2014 11:50:07 GMT+0000 (UTC)
out Wed, 26 Nov 2014 11:50:07 GMT info: To see your app, visit http://localhost:1337
info: To shut down Sails, press + C at any time.

out Wed, 26 Nov 2014 11:50:07 GMT info: /|.\
err Wed, 26 Nov 2014 11:50:07 GMT debug: Environment : development
debug: Port : 1337
debug: --------------------------------------------------------
out Wed, 26 Nov 2014 11:50:07 GMT info: Server lifted in `/opt/run/snapshot/package`
out Wed, 26 Nov 2014 11:50:07 GMT info: .-'.-==|/_--'
out Wed, 26 Nov 2014 11:50:07 GMT info: v0.10.5 |\
out Wed, 26 Nov 2014 11:50:07 GMT info: `--'-------' 
out Wed, 26 Nov 2014 11:50:07 GMT info: Skipping database setup because migrate not set to 'drop'
err Wed, 26 Nov 2014 11:50:07 GMT debug: --------------------------------------------------------
out Wed, 26 Nov 2014 11:50:07 GMT 
out Wed, 26 Nov 2014 11:50:07 GMT info: ,' |' \
err Wed, 26 Nov 2014 11:50:42 GMT error: In model (appliancetype), invalid connection :: mongoAlmanacProd
err Wed, 26 Nov 2014 11:50:42 GMT error: Must contain an `adapter` key referencing the adapter to use.
err Wed, 26 Nov 2014 11:50:51 GMT error: In model (appliancetype), invalid connection :: mongoAlmanacProd
err Wed, 26 Nov 2014 11:50:51 GMT error: Must contain an `adapter` key referencing the adapter to use.
err Wed, 26 Nov 2014 11:51:01 GMT error: In model (appliancetype), invalid connection :: mongoAlmanacProd
err Wed, 26 Nov 2014 11:51:01 GMT error: Must contain an `adapter` key referencing the adapter to use.
err Wed, 26 Nov 2014 11:51:11 GMT error: In model (appliancetype), invalid connection :: mongoAlmanacProd
error: Must contain an `adapter` key referencing the adapter to use.
err Wed, 26 Nov 2014 11:51:21 GMT error: In model (appliancetype), invalid connection :: mongoAlmanacProd
err Wed, 26 Nov 2014 11:51:21 GMT error: Must contain an `adapter` key referencing the adapter to use.

**Comment by CWyrtzen** [comment]
Does this error still exist? Has it been tested with the latest release?

**Comment by sailsbot** [comment]
Thanks for posting, @leejt489.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
